### PR TITLE
Code cleanup and condition changes for score calculations in balanced…

### PR DESCRIPTION
**Pull request by prtsh**
…ResourceScorer, pkg/scheduler

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
this PR changes following (code cleanup, code logic change for early termination and score checks.)
1.  Early termination when the cpuFraction and memoryFraction is >=1
2. Removal of redundant check of cpufraction and memoryfraction when the volumeFraction is >=1
3. Removal of redundant check of cpufraction and memoryfraction when volumeFraction is not included in score calculation

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

Please check the conditions(volume>=1.0 and cpu>=1.0) for early termination of score, condition for score caluclation when volume is included and the final condition when the volume is not included.

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by prtsh** [comment]
/sig scheduling

make update - OK
make test WHAT=./pkg/scheduler - OK

File: pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go [hunk 1]
```diff
@@ -41,10 +41,16 @@ var (
 func balancedResourceScorer(requested, allocable *schedulernodeinfo.Resource, includeVolumes bool, requestedVolumes int, allocatableVolumes int) int64 {
 	cpuFraction := fractionOfCapacity(requested.MilliCPU, allocable.MilliCPU)
 	memoryFraction := fractionOfCapacity(requested.Memory, allocable.Memory)
+
+	if cpuFraction >= 1 || memoryFraction >= 1 {
```
Comment (draveness) [inline pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: This has already been addressed in https://github.com/kubernetes/kubernetes/pull/79453

Comment (draveness) [reply pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: /close #79453 has already get merged.

Thank you anyway for open this PR, @prtsh

**Review by draveness — COMMENTED** [review]
/close

File: pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go (hunk 1 above)
Comment (prtsh) [reply pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: Thanks for the review @draveness

**Pull request closed without merging**